
<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 2.4.0">
  <meta name="generator" content="Hugo 0.51" />
  <meta name="author" content="opencasestudies">

  
  
  
  
    
  
  <meta name="description" content="">

  
  <link rel="alternate" hreflang="en-us" href="/">

  


  

  

  

  
  
  
  <meta name="theme-color" content="#0095eb">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" crossorigin="anonymous">
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href=//fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono>
  

  <link rel="stylesheet" href="/styles.css">
  
  <link rel="stylesheet" href="/css/rladies.css">
  

  
  
  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="opencasestudies">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="opencasestudies">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="opencasestudies">
  <meta property="og:url" content="/">
  <meta property="og:title" content="opencasestudies">
  <meta property="og:description" content="">
  
  
    
  <meta property="og:image" content="/img/icon-192.png">
  <meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2016-04-20T00:00:00-04:00">
  

  

  

  <title>opencasestudies</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71" >

<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/"><img src="/img/ocs-icon.jpg" alt="opencasestudies"></a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav ml-auto">
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#about" data-target="#about">
            
            <span>About</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#casestudies" data-target="#casestudies">
            
            <span>Open Case Studies</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#posts_blog" data-target="#posts_blog">
            
            <span>Blog</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#team" data-target="#team">
            
            <span>Team</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#contact" data-target="#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        

      

        

        
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>


  





  
  
  
  <section id="about" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-address">
  <div class="col-12 col-lg-4">
    <div id="profile">

      
      <img class="portrait" src="/img/ocs-icon.jpg" itemprop="image">
      

      <div class="portrait-title">
        <h2 itemprop="name">opencasestudies</h2>
        

        
        

        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          
          <span itemprop="name"></span>
          
        </h3>
        
      </div>

      <link itemprop="url" href="/">

      <ul class="network-icon" aria-hidden="true">
        
        
        
        
          
        
        <li>
          <a itemprop="sameAs" href="mailto:shicks19@jhu.edu" target="_blank" rel="noopener">
            <i class="fas fa-envelope big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        <li>
          <a itemprop="sameAs" href="//github.com/opencasestudies" target="_blank" rel="noopener">
            <i class="fab fa-github big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-12 col-lg-8" itemprop="description">

    

<h1 id="the-ocs-project">The OCS project</h1>

<h2 id="what-problem-are-we-addressing">What problem are we addressing?</h2>

<p>An increase in demand for gateway training in statistics, biostatistics, and data science has led to changes in curriculum, specifically an increase in computing. While this has led to more applied courses, students still struggle with effectively deriving knowledge from data. This is primarily because (i) these courses frequently fail to frame the lectures around a real-world application; (ii) quantitative methods are typically illustrated with an unrealistically clean data set that fits the assumptions of the method in an equally unrealistic way. When students use this approach to solve problems in the real-world, they are unable to identify the most appropriate methodological approach when it is not spoon fed.</p>

<h2 id="problems-with-previously-suggested-solutions">Problems with previously suggested solutions</h2>

<p>In 1999, <a href="https://www.stat.berkeley.edu/users/statlabs/" target="_blank">Nolan and Speed</a> argued the solution was to teach courses through in-depth case studies derived from interesting problems, with nontrivial solutions that leave room for different analyses. This innovative framework teaches the student to make important connections between the scientific question, data and statistical concepts that only come from hands-on experience analyzing data. However, these case studies based on realistic challenges, not toy examples, are scarce.</p>

<h2 id="what-are-we-proposing-as-a-solution">What are we proposing as a solution?</h2>

<p>To address this, we are developing the <code>opencasestudies</code> educational resource of case studies, which demonstrate illustrative data analyses that can be used in the classroom to teach students how to effectively derive knowledge from data. This approach has successfully been used to <a href="https://amstat.tandfonline.com/doi/abs/10.1080/00031305.2017.1356747#.XDZCzS3MxTY" target="_blank">teach data science</a> courses at many universities, including:</p>

<ul>
<li><a href="https://jhu-advdatasci.github.io/2018/" target="_blank">Johns Hopkins Bloomberg School of Public Health</a></li>
<li><a href="http://cs109.github.io/2014/" target="_blank">Harvard University</a></li>
<li><a href="http://datasciencelabs.github.io/2016/" target="_blank">Harvard T.H. Chan School of Public Health</a></li>
<li><a href="http://rdatasciencecases.org" target="_blank">University of California at Berkeley</a></li>
<li><a href="https://www.tandfonline.com/doi/pdf/10.1080/00031305.2015.1081105" target="_blank">Smith College</a></li>
</ul>


    <div class="row">

      

      

    </div>
  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="casestudies" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-lg-12">
    

<h1 id="open-case-studies">Open Case Studies</h1>

<h2 id="what-do-we-mean-by-a-case-study">What do we mean by a <em>case study</em> ?</h2>

<p>Here, we define the term <em>case study</em> as an illustrative, complete data analysis in a real-world application, such as addressing obesity and risks to adolescent health:</p>

<ol>
<li>Begins with an interesting scientific question or application for motivation.

<ul>
<li><em>Can we better identify patients in young, minority populations at risk for diabetes?</em></li>
</ul></li>
<li>Discusses how to access, gather, import, clean, and visualize data.

<ul>
<li><em>Data from electronic health records</em></li>
</ul></li>
<li>Exposes students to the thought process of combining computational and programming aspects with statistical concepts in a concrete example of data analysis. This is different from teaching either a programing language or a statistical methods class individually.

<ul>
<li><em>How can I use Monte Carlo simulations to simulate health outcomes? What does &lsquo;relative risk&rsquo; mean? Can I use machine learning to predict outcomes?</em></li>
</ul></li>
<li>Illustrates nontrivial solutions that leave room for different analyses of the data.

<ul>
<li><em>Are there possible biases to account for in the prediction model (e.g. selection bias, geographic information)?</em></li>
</ul></li>
<li>Teaches the student to effectively communicate results and to make connections between the scientific question, data and statistical concepts that only come from hands-on experience analyzing data.</li>
</ol>

<h2 id="available-open-case-studies">Available Open Case Studies</h2>

<p>We are actively working on creating case studies and this is
the current list of available case studies:</p>

<ul>
<li><p>Exploring health expenditure in the United States:  [<a href="https://github.com/opencasestudies/ocs-healthexpenditure/blob/master/README.md" target="_blank">description</a>] [<a href="casestudies/ocs-healthexpenditure.html" target="_blank">case study</a>] [<a href="https://github.com/opencasestudies/ocs-healthexpenditure" target="_blank">code</a>]</p></li>

<li><p>Examine relationship between fatal police shootings and firearm legislation in the United States:  [<a href="https://github.com/opencasestudies/ocs-police-shootings-firearm-legislation/blob/master/README.md" target="_blank">description</a>] [<a href="casestudies/ocs-police-shootings-firearm-legislation.html" target="_blank">case study</a>] [<a href="https://github.com/opencasestudies/ocs-police-shootings-firearm-legislation" target="_blank">code</a>]</p></li>
</ul>

<p>If you would like to contribute a case study to the repository,
or have suggestions on data sets to use to build a case study around,
please contact us!</p>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="posts_blog" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-12 col-lg-4 section-heading">

    <h1>Recent Blog Posts</h1>
    
    

  </div>
  <div class="col-12 col-lg-8">

    

    
      

      
      
    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="team" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-12 col-lg-4 section-heading">
    <h1>Team</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

<h1 id="meet-our-ocs-team">Meet our OCS team</h1>

<h4 id="stephanie-hicks">Stephanie Hicks</h4>

<p><img src="/img/stephanie_hicks.jpg" style="width:20%";/></p>

<p><a href="http://www.stephaniehicks.com" target="_blank">Stephanie</a> is an
Assistant Professor at
<a href="https://www.jhsph.edu" target="_blank">Johns Hopkins Bloomberg School of Public Health</a>.</p>

<h4 id="leah-jager">Leah Jager</h4>

<p><img src="/img/leah_jager.jpg" style="width:20%";/></p>

<p>Leah is an Assistant Scientist in the <a href="https://www.jhsph.edu/departments/biostatistics/index.html" target="_blank">Biostatistics Department</a>
at <a href="https://www.jhsph.edu" target="_blank">Johns Hopkins Bloomberg School of Public Health</a>.</p>

<h4 id="margaret-taub">Margaret Taub</h4>

<p><img src="/img/margaret_taub.jpg" style="width:20%";/></p>

<p>Margaret is an Assistant Scientist in the
<a href="https://www.jhsph.edu/departments/biostatistics/index.html" target="_blank">Biostatistics Department</a>
at <a href="https://www.jhsph.edu" target="_blank">Johns Hopkins Bloomberg School of Public Health</a>.</p>

<h4 id="alex-stephens">Alex Stephens</h4>

<h4 id="ted-zhang">Ted Zhang</h4>

<h4 id="perry-kuo">Perry Kuo</h4>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="contact" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Contact</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

    <ul class="fa-ul" itemscope>

      
      <li>
        <i class="fa-li fas fa-envelope fa-2x" aria-hidden="true"></i>
        <span id="person-email" itemprop="email"><a href="mailto:shicks19@jhu.edu">shicks19@jhu.edu</a></span>
      </li>
      

      

      

      

      

      

      
      <li>
        <i class="fa-li fas fa-map-marker fa-2x" aria-hidden="true"></i>
        <span id="person-address" itemprop="address">Baltimore, USA</span>
      </li>
      

      

    </ul>

    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="tags" class="home-section">
    <div class="container">
      


<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Tags</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="search" class="home-section">
    <div class="container">
      


<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Search</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    <div id="search-box">
      
      <input name="q" id="search-query" placeholder="Search..." autocapitalize="off" autocomplete="off" autocorrect="off" role="textbox" spellcheck="false" type="search">
      
    </div>

    <div id="search-hits">
      
    </div>

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="projects" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-12 col-lg-4 section-heading">
    <h1>Projects</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

<h3 id="analysis-of-genomic-data-from-single-cells">Analysis of genomic data from single cells</h3>

<p>Single-cell RNA-Sequencing (scRNA-seq) data has become the most widely used high-throughput method for transcription profiling of individual cells. This technology has created an unprecedented opportunity to investigate important biological questions that can only be answered at the single-cell level. However, this technology also brings new statistical, computational and methodological challenges.</p>

<h4 id="methods-to-address-technical-variablity">Methods to address technical variablity</h4>

<p>In contrast to bulk RNA-seq experiments, the majority of reported expression levels in scRNA-seq data are zeros, which could be either biologically-driven, genes not expressing RNA at the time of measurement, or technically-driven, genes expressing RNA, but not at a sufficient level to be detected by sequencing technology. In addition, systematic errors, including batch effects, have been widely reported as a major challenge in high-throughput technologies, however, surprisingly, these issues have received minimal attention in published studies based on scRNA-seq technology. To investigate this, I examined data from fifteen published scRNA-seq studies and demonstrated that systematic errors can explain a substantial percentage of observed cell-to-cell expression variability, which in turn can lead to false discoveries, for example, when using unsupervised learning methods (1). More recently, we developed a dimensionality reduction method, Varying-censoring Aware Matrix Factorization (VAMF), which permits the identification of low-dimensional representations of cells in the presence of cell-specific censoring. This allows for the correction for batch effects if they are mediated through a varying censoring mechanism in either confounded or unconfounded study designs, which is not possible using standard batch correction methods (2).</p>

<ol>
<li>Hicks SC, Townes FW, Teng M, Irizarry RA (2018). <a href="https://academic.oup.com/biostatistics/article/19/4/562/4599254" target="_blank">Missing data and technical variability in single-cell RNA-sequencing experiments</a>. <em>Biostatistics</em>.</li>
<li>Townes FW, Hicks SC, Aryee MJ, Irizarry RA (2017). <a href="https://www.biorxiv.org/content/early/2017/07/21/166736" target="_blank">Varying-Censoring Aware Matrix Factorization for Single Cell RNA-Sequencing</a>. <em>bioRxiv</em>.</li>
</ol>

<h4 id="fast-scalable-memory-efficient-methods-to-analyze-single-cell-data">Fast, scalable, memory-efficient methods to analyze single-cell data</h4>

<p>The <em>k</em>-means algorithm is a classic algorithm used in the analysis of scRNA-seq data. However, with increasing sizes of single-cell data, new methods are needed that are fast, scalable and memory-efficient. To address this, we are implementing the mini-batch optimization for <em>k</em>-means clustering proposed in <a href="https://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf" target="_blank">Sculley (2010)</a> for large single cell sequencing data (1). The mini batch <em>k</em>-means algorithm can be run with data stored in memory or on disk (e.g. HDF5 file format).</p>

<ol>
<li><a href="https://github.com/drisso/mbkmeans" target="_blank">mbkmeans</a>.  Mini-batch k-means clustering for large single-cell datasets.</li>
</ol>

<h4 id="high-grade-serous-ovarian-cancer-subtypes-with-single-cell-profiling">High-grade serous ovarian cancer subtypes with single-cell profiling</h4>

<p>The goal of this project is to identify the biological basis of subtypes of high-grade serous ovarian cancers (HGSOC) using bulk and single-cell gene expression data. This is highly relevant to public health because HGSOC is a particularly deadly cancer that is often only identified at late stage and treatment options are limited. The long-term impact of this project will be a key step towards developing targeted treatments for HGSOCs.</p>

<p><strong>If you are interested in this project</strong>, there is an open postdoctoral scientist position (see the <a href="../join/index.html">Join us</a> page for more information)!</p>

<hr />

<h3 id="data-science-education">Data Science Education</h3>

<p>An increase in demand for statistics and data science education has led to changes in curriculum, specifically an increase in computing. While this has led to more applied courses, students still struggle with effectively deriving knowledge from data and solving real-world problems. In 1999, Deborah Nolan and Terry Speed argued the solution was to teach courses through in-depth case studies derived from interesting scientific questions with nontrivial solutions that leave room for different analyses of the data. This innovative framework teaches the student to make important connections between the scientific question, data and statistical concepts that only come from hands-on experience analyzing data (1, 2). To address this, I am building the <a href="https://github.com/opencasestudies" target="_blank">openDataCases</a> community resource of case studies that educators can use in the classroom to teach students how to effectively derive knowledge from data.</p>

<ol>
<li>Hicks SC, Irizarry RA (2018). <a href="https://www.tandfonline.com/doi/abs/10.1080/00031305.2017.1356747?journalCode=utas20" target="_blank">A Guide to Teaching Data Science</a>. <em>The American Statistician</em>.</li>
<li>Hicks SC (2017). <a href="https://www.tandfonline.com/doi/abs/10.1080/10618600.2017.1385472" target="_blank">Greater Data Science Ahead</a>. <em>Journal of Computational Graphical Statistics</em>.</li>
</ol>

<hr />

<h3 id="statistical-methods-to-control-for-false-discoveries">Statistical methods to control for false discoveries</h3>

<p>In high-throughput studies, hundreds to millions of hypotheses are typically tested. Statistical methods that control the false discovery rate (FDR) have emerged as popular and powerful tools for error rate control. While classic FDR methods use only <em>p</em>-values as input, more modern FDR methods have been shown to increase power by incorporating complementary information as &ldquo;informative covariates&rdquo; to prioritize, weight, and group hypotheses. To address this, we investigated the accuracy, applicability, and ease of use of two classic and six modern FDR-controlling methods by performing a systematic benchmark comparison using simulation studies as well as six case studies in computational biology (1).</p>

<ol>
<li>Korthauer K, Kimes PK, Duvallet C, Reyes A, Subramanian A, Teng M, Shukla C, Alm EJ, Hicks SC (2018). <a href="https://doi.org/10.1101/458786" target="_blank">A practical guide to methods controlling false discoveries in computational biology</a>. <em>bioRxiv</em>.</li>
</ol>

<hr />

<h3 id="statistical-methods-for-normalization-of-high-throughput-data">Statistical methods for normalization of high-throughput data</h3>

<p>Normalization is an essential step for the analysis of genomics high-throughput data. Quantile normalization is one of the most widely used multi-sample normalization tools for applications including genotyping arrays, RNA-Sequencing (RNA-Seq), DNA methylation, ChIP-Sequencing (ChIP-Seq) and brain imaging. However, quantile normalization relies on assumptions about the data-generation process that are not appropriate in some contexts. I developed a data-driven method to test these assumptions and guide the choice of an appropriate normalization method (1). The <a href="https://bioconductor.org/packages/release/bioc/html/quantro.html" target="_blank">freely available software</a> has been downloaded over 7500 times (distinct IPs) from Bioconductor since 2014 and has helped researchers test the assumptions of global normalization methods in the analysis of their own data. To address the scenario when the assumptions of quantile normalization are not appropriate, I have developed a generalization of quantile normalization, referred to as smooth quantile normalization, which allows for global differences between biological groups (2). More recently, I collaborated with researchers from the University of Maryland to correct for compositional biases found in sparse metagenomic sequencing data (3).</p>

<ol>
<li>Hicks SC, Irizarry RA (2015). <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0679-0" target="_blank">quantro: a data-driven approach to guide the choice of an appropriate normalization method</a>. <em>Genome Biology</em>.</li>
<li>Hicks SC, Okrah K, Paulson JN, Quackenbush J, Irizarry RA, Bravo HC (2018). <a href="https://academic.oup.com/biostatistics/article-abstract/19/2/185/3949169?redirectedFrom=fulltext" target="_blank">Smooth quantile normalization</a>. <em>Biostatistics</em>.</li>
<li>Kumar MS, Slud EV, Okrah K, Hicks SC, Hannenhalli S, Corrada Bravo H (2018). <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5160-5" target="_blank">Analysis and correction of compositional bias in sparse sequencing count data</a>. <em>BMC Genomics</em>.</li>
</ol>

  </div>
  
</div>

    </div>
  </section>
  



<div class="container">
  <footer class="site-footer">
  

  <p class="powered-by">
    &copy; 2019 opencasestudies under (CC) BY-NC-SA 4.0. &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

</div>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha512-+NqPlbbtM1QqiK8ZAo4Yrj2c4lNQoGv8P79DPtKzj++l5jnN39rHA/xsqn8zE9l0uSoxaCdrOgFs6yjyfbBxSg==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha256-VsEqElsCHSGmnmHXGQzvoWjWwoznFSZc6hs7ARLRacQ=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
        
        <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
        
        <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/bash.min.js"></script>
        
        <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
        
        <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/stata.min.js"></script>
        
      

      
      
    

    <script src="/js/hugo-academic.js"></script>
    

    
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "Search Results",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    <script src="/js/search.js"></script>
    

    
    

  </body>
</html>


